app/components/action-feed/component.js

```diff
@@ -12,15 +12,53 @@ import { translationMacro as t } from 'ember-i18n';
  * ```
  * @class action-feed
  */
+
 export default Component.extend({
     store: service(),
     toast: service(),
-
+    currentUser: service(),
     classNames: ['action-feed'],
+    page: 1,
 
     errorMessage: t('components.action-feed.error_loading'),
 
     moreActions: computed('totalPages', 'page', function() {
         return this.get('page') < this.get('totalPages');
     }),
+
+    init() {
+        this._super(...arguments);
+        this.set('listLoading', true);
+        this.set('actionsList', []);
+        this.loadPage();
+    },
+
+    loadPage() {
+        const page = this.get('page');
+        this.get('currentUser.user')
+            .then(user => this.get('store').queryHasMany(user, 'actions', { page }))
+            .then(this._setPageProperties.bind(this))
+            .catch(this._handleLoadError.bind(this));
+    },
+
+    _handleLoadError() {
+        this.set('listLoading', false);
+        this.set('loadingMore', false);
+        this.get('toast').error(this.get('errorMessage'));
+    },
+
+    _setPageProperties(response) {
+        this.setProperties({
+            actionsList: this.get('actionsList').pushObjects(response.toArray()),
```

Comment:
`pushObjects` modifies the array in place, this is just setting `actionsList = this.get('actionsList')`. It'd make more sense to do that one line up:
```js
this.get('actionsList').pushObjects(...);
this.setProperties({
    totalPages: ...
});